libraries/AP_Math/AP_Math.h

```diff
@@ -212,7 +212,17 @@ inline double constrain_double(const double amt, const double low, const double
 }
 
 // degrees -> radians
-static inline constexpr ftype radians(ftype deg)
+static inline constexpr double radians(double deg)
```

Comment:
```suggestion
static inline constexpr double radians(double deg) WARN_IF_UNUSED
```
Prevent bugs?

Reply:
I don't understand the value, e.g. `sin` is not tagged this way and we have survived this far without that.

Reply:
No runtime cost and prevent bugs. There's no downside other than having to read it in the header.